src/components/material/StockAndReservationInfo.tsx

```diff
@@ -22,10 +24,17 @@ const StockAndReservationInfo: FC<StockAndReservationInfoProps> = ({
     placeholders: { "@count": reservationCount }
   });
 
+  const numberInQueueText = numberInQueue
+    ? t("numberInQueueText", {
+        placeholders: { "@number": numberInQueue }
+      })
+    : false;
+
   return (
     <>
-      {materialsInStockInfoText && `${materialsInStockInfoText}.`}{" "}
-      {materialReservationInfoText && materialReservationInfoText}
+      {materialsInStockInfoText && `${materialsInStockInfoText}.`}
+      {materialReservationInfoText && ` ${materialReservationInfoText}.`}
```

Comment:
Is it necessary with template literals here? If yes, I don't think the leading space is needed.